README.md

```diff
@@ -2,28 +2,31 @@
 
 ## Test & Lint Workflow
 
-```
+```yml
 name: Test & Lint
 on: push
 
 jobs:
   do:
-    uses: parachutehome/workflows/.github/workflows/jest-test-and-lint.yml@v1
```

Comment:
Only think I think we'll want to do is make a `v2` release with these changes. We should be able to do so with the `https://github.com/parachutehome/workflows/blob/main/set-version.sh` script easily enough though.